starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,49 @@
+function main () {
+  const coinDeskApi = axios.create({
+    baseURL: 'http://api.coindesk.com/v1/bpi/historical/close.json'
+  })
+
+  function getCoinInfo(id) {
+  coinDeskApi.get(id)
+  .then(response => {
+    console.log(response.data.bpi);
+    var p = response.data.bpi;
+    var date;
+    var price;
+    var ourData = [];
+
+    for (var key in p) {
+      if (p.hasOwnProperty(key)) {
+
+        // ourData.push({
+        //   x: key,
+        //   y: p[key]
+        // });
+        ourData.push(p[key]);
```

Comment:
There is js method for getting an array of the object's keys. Also there is a method  for getting the object's values.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values